portal-web/test/functional/com/liferay/portalweb/macros/GogoShell.macro

```diff
@@ -37,7 +37,16 @@ definition {
 	}
 
 	macro openGogoShellAdmin {
-		Navigator.openWithAppendToBaseURL(urlAppend = "group/control_panel/manage?p_p_id=com_liferay_gogo_shell_web_internal_portlet_GogoShellPortlet");
+		if (isSet(baseURL)) {
+			var baseURL = ${baseURL};
```

Comment:
@catarineaguiar Why is the variable set again here if it is already set?

Reply:
I got this part from other macro and didn't ask me why it was duplicating the variable. I'll send another pull without it.